core-services/prow/02_config/_plugins.yaml

```diff
@@ -945,6 +954,17 @@ config_updater:
         build02:
         - ci
       name: prow-job-master-sidecar-4.4
+    ci-operator/templates/openshift/crc/crc-launch-e2e.yaml:
+      clusters:
+        api.ci:
+        - ci
+        app.ci:
+        - ci
+        build01:
+        - ci
+        build02:
+        - ci
+      name: prow-job-crc-launch-e2e
```

Comment:
This hunk needs to be merged in a separate PR first to make sure the template is correctly propagated to the clusters when *this* PR merges.